ci_watson/hst_helpers.py

```diff
@@ -108,6 +116,8 @@ def download_crds(refname, timeout=30, verbose=False):
 
     """
     refdir = None
+    fname = refname
+    print('Deprecation Warning: `timeout` parameter no longer needed.')
```

Comment:
It is a bit brute force to print this regardless. You could change the signature to take `**kwargs` and only emit warning if `timeout` is given. Also better to use `warnings.warn(..., DeprecationWarning)`.

Reply:
Good suggestion.  Coming right up...

Reply:
Changed to using 'warnings.warn' in [e00d94c](https://github.com/spacetelescope/ci_watson/pull/53/commits/e00d94c5ff026ff1ee9658d797658617583823be)